tlspuffin/src/tls/rustls/msgs/handshake_test.rs

```diff
@@ -462,7 +462,7 @@ fn test_truncated_client_extension_is_detected() {
 
     for ext in &chp.extensions.0 {
         let mut enc = ext.get_encoding();
-        // println!("testing {:?} enc {:?}", ext, enc);
+        // log::trace!("testing {:?} enc {:?}", ext, enc);
```

Comment:
should we keep this comment ?

Reply:
I kept it to make sure one can resurrect logging in the future should one want to.